src/utils/PokeApi.jsx

```diff
@@ -0,0 +1,62 @@
+export const baseUrl = "https://pokeapi.co/api/v2/pokemon";
+
+export async function fetchPokemon() {
+  function arrangeTypes(data) {
+    const typeHolder = document.getElementById("pokeType");
```

Comment:
`document.querySelector` or `getElementById` should **not be used** in `React` because it’s a **very slow** operation for the browser. React was specially created not to search anything in the DOM. So, you can directly operate with the markup inside the components. Or you can use hook `useRef` for the reference.


But you should `pass` the type of the pokemon into `fetchPokemon`